pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -1596,6 +1596,28 @@ Other functions
   If this function exists, it is called every second to do regular tasks.
   This can be used for e.g. :doc:`Dynamic Blocks <../guides/dynblocks>`.
 
+.. function:: threadmessage(cmd, dict)
+
+  .. versionadded:: 1.7.0
+
+  This function, if it exists, is called when a separate thread (made with :func:`newThread`) calls :func:`submitToMainThread`.
```

Comment:
What happens if `threadmessage` does not exist and `submitToMainThread` is called?

Reply:
`      errlog("Lua thread called submitToMainThread but no threadmessage receiver is defined");`

Reply:
I think it should be documented if it is a no-op or error. Current PR leaves that open.

Reply:
will do

Reply:
amended